### DISCUSSION TITLE
Selecting a stripping script for Windows 10 pro for Winapps image

### DISCUSSION THREAD

**Discussion by freechelmi** [winapps-org/winapps#9] [General]
I'm not really a windows expert ...

We need to select " strip & slim" script ( maybe PS ) that will generate a new ISO from official Windows 10 Pro iso which could be used for the winapps image generation 

Currently I know : 

- TIny10 : RDP does not work AFAIK
- https://ameliorated.io : Nice but not so light 

I guess we could target a 10 GB installed windows image

**Comment by oskardotglobal** [comment]
[Ameliorated has a variant called "ReviOS"](https://www.revi.cc/revios/download) which is basically configuration for the image creator ameliorated offers. They have ISO downloads on their website.
Alternatively, we build our own scripts based on https://github.com/ChrisTitusTech/winutil (I don't think we can use it out of the box since it opens up a UI)

edit: bad link

**Reply by oskardotglobal** [reply]
Ok, I've taken a second look at this and concluded we cannot include the ISO. We can probably link to the websites of tiny10 & ameliorated, but I wouldn't include direct links to the files. Another thing we'll need is the user to activate windows 10 pro by hand, since I don't think we should include one of those "activation scripts" in winapps. 

[Also, some tiny10 versions support RDP](https://www.reddit.com/r/tiny10/comments/v5su5h/comment/jnfanmk/?utm_source=share&utm_medium=web2x&context=3), but all the download links are pure jank too, so best if we build ameliorated images ourselves and distribute them seperatly

**Reply by freechelmi** [reply]
Yes , the windows ISO subject is not an easy one.

But I guess it is a low priority at first as most of our testers will know how to get the iso or slim it 

Let's keep the discussion open but the ameliorated scripts ( like atlas ) seems to be one of the best solution

**Comment by freechelmi** [comment]
Tiny11 has a video that explain how it strip things , could be a good starting point if we keep RDP 

https://www.youtube.com/watch?v=eYIDQLavZ7U
